react/src/lib/Sidebar/index.js

```diff
@@ -48,14 +49,17 @@ class Sidebar extends React.Component {
 
   render() {
     const {
+      buttonProps,
       children,
       className,
-      expandable,
+      collapsable,
```

Comment:
does this mean you can turn off the collapsing?  or should the variable be collapsed??  mean it's collapsed or not?

Reply:
This prop is to turn on/off collapsing. i.e. on momentum.design, the sidebar is not collapsable but in Atlas it is.